school_library_people.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @id = 1
+  end
+
+  # rubocop:disable Naming/PredicateName
+  def is_of_age?
+    @age > 18
+  end
+  # rubocop:enable Naming/PredicateName
+  private :is_of_age?
```

Comment:
- [OPTIONAL] It is a good practice to set the private keyword on its own line and all other functions below it are automatically taken to be private methods.

See example:
```ruby
private

def testone; end
def testtwo; end

```


From the example, testone and testtwo are private methods.